Sources/Growth/URP/UrpService.swift

```diff
@@ -96,11 +102,35 @@ struct UrpService {
         
         return adAttributionData
       }
+      
+      throw SearchAdError.invalidCampaignTokenData
+    } catch {
+      throw error
+    }
+  }
+  
+  @MainActor func adGroupReportsKeywordLookup(adGroupId: Int, campaignId: Int, keywordId: Int) async throws -> String {
+    let reportsURL = adReportsURL + "campaigns/\(campaignId)/adgroups/\(adGroupId)/keywords"
```

Comment:
Use `append(pathComponents: String...)` instead of string interpolation maybe?

Reply:
Changed with https://github.com/brave/brave-ios/pull/8591/commits/b923721b1e38e626aa1c8c41430c83f3433580b8